src/tests/multihost/ipa/conftest.py

```diff
@@ -97,6 +138,28 @@ def remove_ipa_users():
     request.addfinalizer(remove_ipa_users)
 
 
+@pytest.fixture(scope="class")
+def default_ipa_groups(session_multihost, request):
```

Comment:
I think that the fixture should provide a dictionary with the group names as keys and gids as values for the tests to use.